process/ReleaseProcess.md

```diff
@@ -0,0 +1,159 @@
+# Release Process and Guidelines
+
+## What is a Zowe release
+
+A release of the Zowe project consists of two 'convenience' deliverables - CLI binaries, and a PAX file which contains the Zowe API Mediation Layer, Zowe Desktop, and Zowe Explorer-Server. The CLI binaries are obtainable either through standard npm install commands, or, for users with limited internet access, through a convience zip file which allows for offline installation. The CLI convenience zip and Zowe PAX are both available through the official [Zowe Website](https://www.zowe.org).
+
+Each Zowe release comes with updates to user installation, configuration, and extension documentation. The documentation is developed in the [docs-site repository](https://github.com/zowe/docs-site), is accessible through the [Zowe Website](https://www.zowe.org), and is hosted directly on [Github.io](https://zowe.github.io/docs-site/).
+
+The focus of this document refers to generating stable releases of Zowe. For bleeding edge Zowe builds, see [Zowe Nightly Builds](#zowe-nightly-builds).
+
+Generating any release of Zowe requires gathering artifacts from each of Zowe's subprojects, organizing them into the Zowe deliverables, testing said Zowe deliverables, signing the deliverables, and publishing the deliverables in sync with Release Notes / Documentation Updates.
+
+## How to generate a release
+
+Stable Zowe releases are generated from [zowe-install-packaging](https://github.com/zowe/zowe-install-packaging) repository **master** branch. The versions defined in [zowe-install-packaging](https://github.com/zowe/zowe-install-packaging) are Zowe official stable version.
+
+In order to generate a new release of Zowe, each of the below checklists must be completed. The checklists below are roughly ordered based on prerequisite resolution. 
+
+### Subproject Checklist.
+  
+* Each subproject **must** confirm the version(s) of their planned deliverable artifacts. **Official Zowe deliverables must not depend on SNAPSHOTS or other imprecise artifact versioning schemas**. For details, see [Subproject Release Cadence](#subproject-release-cadence).
+* Each subproject **must** identify if any of their dependencies have changed since the last release.
+* Each subproject **must** create release notes for their project. For details, see [Creating Release Notes](#creating-release-notes)
+
+### Zowe CLI Convenience Bundle Checklist
+
+* Confirm @next versions of the core CLI as well as plugins for packaging.
+
+### Zowe Documentation Checklist
+
+* Aggregate release notes from the Zowe subprojects, and place them in the [docs-site repository](https://github.com/zowe/docs-site).
+* Merge release notes and any other docs-site feature branches into the [docs-staging branch](https://github.com/zowe/docs-site/tree/docs-staging).
+
+### Zowe PAX Checklist
```

Comment:
Just making a note this is related to the binary release question from above..

Reply:
I've used the term convenience build to disambiguate from source.  Is there a better term?